### DISCUSSION TITLE
Faster Whisper  always fails on Windows 11.

### DISCUSSION THREAD

**Discussion by galaxea** [chidiwilliams/buzz#535] [General]
Hello,

So far, I have not been able to get Faster Whisper to work. I downloaded the models and attempted to use different sizes but they never work. The normal models do work. Any idea why this would happen? Thanks for any assistance on this.

**Comment by kiya52** [comment]
/Dear chidiwilliams,

     I met the same situation. The newest v.1.0.1 still has the same problem. It doesn't works by useing fast whisper model.
     Should I install the geforce 1650 newest drivers? which version is necessary?
     notebook: acer nitro 5, i7 10th gen, geforce 1650( driver: game ready 555.85)

**Comment by raivisdejus** [comment]
Please see if selecting language to transcribe to helps. One bug in version 1.0.1 will cause crash if "Detect language" is selected. Workaround for this is to select a language to transcribe to.

**Comment by Code-1123** [comment]
Me too! 'Unknown error' shows when Faster Whispe runs

**Comment by raivisdejus** [comment]
Please try latest development version from https://github.com/chidiwilliams/buzz/actions/runs/10211346492
You have to log into github to see the artifacts section

**Comment by Zsural** [comment]
Can you add a version with requirements of "ctranslate2==3.24.0 + faster-whisper 0.10.0", because it supports cuda11 and cublas64_11, And from ctranslate2>=4.0.0,it only supports cuda12

**Reply by raivisdejus** [reply]
Keeping track of current version on multiple OSes is challenging enough, so I would not want to have another variant to support.
But maybe we can switch to the older version as default. At some free moment will test if this setup works fine on CUDA 12 and if we are loosing some speed with the older version.

**Reply by Zsural** [reply]
Thanks to your great work! From the version 0.8.2, I can' t use faster-whisper to transcribe the first time,but now I still can't use it. I think it's better to mention it by myself. Does this repo contain all entire code of Buzz(Window)? I just want to replace or adjust the dependencies and wont' to mention much code, or can you give me some advice?  Thanks

**Reply by raivisdejus** [reply]
Yes all code is here. See notes on running locally here https://github.com/chidiwilliams/buzz/blob/main/CONTRIBUTING.md

The hard part with windows is all the dependencies needed to run builds on whisper.cpp, but if you copy the DLLs from the backup folder you can skip that part and adjust the dependencies in poetry.lock file or use [poetry commadns](https://python-poetry.org/docs/cli/) to do that. So should be relatively easy to adjust.

buzz.exe is a package of python and all the libraries it needs, but you can run Buzz in a python / source mode with `python -m buzz`

Please share results of your tests, did it work, did it have any errors?

**Reply by Zsural** [reply]
ok!Thanks

**Reply by raivisdejus** [reply]
@Zsural Unfortunately downgrading Faster whisper is not an option, it causes build errors due to seeming `av` incompatibility with `python-12`

If you are on older python version try downgrading faster whisper locally.

**Comment by raivisdejus** [comment]
Please try the latest version 1.2.0 https://github.com/chidiwilliams/buzz/releases

**Comment by Voltius-PC** [comment]
Nope. I'm on the latest version and downloaded the faster-whisper Large v3 turbo and it just doesn't work.

**Reply by raivisdejus** [reply]
@Voltius-PC Please try the version 1.3.3 https://github.com/chidiwilliams/buzz/releases If it still fails, please let me know what GPU you have if any. We have some yet unresolved issues with Faster whisper on the very latest Nvidia GPUs. If that is your case, you can try some other whisper type. With a fast GPU all of them should work similarly fast.
